.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,23 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.128.0/containers/jekyll
+{
+  "name": "Jekyll",
+  "dockerFile": "Dockerfile",
+  // Set *default* container specific settings.json values on container create.
+  "settings": {
+    "terminal.integrated.shell.linux": "/bin/bash"
+  },
+  // Add the IDs of extensions you want installed when the container is created.
+  "extensions": [
+    "davidanson.vscode-markdownlint",
+    "editorconfig.editorconfig"
+  ],
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  "forwardPorts": [
+    4000
+  ],
+  // Use 'postCreateCommand' to run commands after the container is created.
+  "postCreateCommand": "cd ${input:projectName} && bundle install",
```

Comment:
Should we remove `bundle install` instruction from the WIKI then if we have it here?

Reply:
Hm... interesting. The `bundle install` from the `postCreateCommand` doesn't seem to execute because after the container has started I cannot start jekyll, due to its dependencies not being installed. 
I'll need to look into that as well.

Reply:
@zvirja the `bundle install` now works on container startup, no need to install the dependencies manually. I've also removed the statement from the wiki.